test/harvest/test_dimensions.py

```diff
@@ -43,22 +34,125 @@ def test_publication_fields():
     assert "title" in fields
 
 
-def test_publications_csv(tmpdir):
-    pubs_csv = tmpdir / "dimensions-pubs.csv"
-    dimensions.publications_csv(
-        ["10.48550/arxiv.1706.03762", "10.1145/3442188.3445922"], pubs_csv
-    )
-
-    df = pandas.read_csv(pubs_csv)
-
-    assert len(df) == 2
-
-    # the order of the results isn't guaranteed but make sure things are coming back
+def test_orcid_publications():
```

Comment:
another live API test to make sure that our querying and result wrangling continues to work